packages/route-viewer-overlay/src/index.tsx

```diff
@@ -58,33 +58,51 @@ const RouteViewerOverlay = (props: Props): JSX.Element => {
   const { routeData } = props;
   useEffect(() => {
     // if pattern geometry updated, update the map points
+    let bounds;
     if (isGeometryComplete(routeData)) {
       const allPoints: LngLatLike[] = Object.values(routeData.patterns).reduce(
         (acc, ptn) => {
           return acc.concat(polyline.decode(ptn.geometry.points));
         },
         []
       );
+
       if (allPoints.length > 0) {
-        const geoJsonedPoints: [number, number][] = allPoints.map(c => [
-          c[1],
-          c[0]
-        ]);
-        const bounds = geoJsonedPoints.reduce((bnds, coord) => {
+        const geoJsonedPoints: [number, number][] = allPoints.map(c => {
+          return [c[1], c[0]];
+        });
+        bounds = geoJsonedPoints.reduce((bnds, coord) => {
           return bnds.extend(coord);
         }, new LngLatBounds(geoJsonedPoints[0], geoJsonedPoints[0]));
+      }
+    }
 
-        current?.fitBounds(bounds, {
-          duration: 500,
-          padding: window.innerWidth > 500 ? 200 : undefined
-        });
+    // if pattern geometry updated, update the map points
+    if (bounds && current) {
+      const canvas = current.getCanvas();
+      // @ts-expect-error getOixelRatio not defeined in MapRef type.
```

Comment:
Typo!

Reply:
Fixed in https://github.com/opentripplanner/otp-ui/pull/485/commits/fb1c25df7b28b974859562e58e13e0e468377953.